index.html

```diff
@@ -32,6 +32,106 @@ <h1 id="name" class="text">
+              </ul>
+            </div>
+            <div type="image" id="prdetailbutton"></div>
+          </div>
+          <div id="pr03">
+            <img id="pr03img" src="images/pr03img.png">
+            <div class="projectinfo" id="pr_info_03">
+              <div class="projectname"><h2>Project name goes here</h2></div>
+              <div>
+                <ul class="techtags">
+                  <li class="techtag">HTML</li>
+                  <li class="techtag">CSS</li>
+                  <li class="techtag">Javascript</li>
+                </ul>
+              </div>
+              <div type="image" id="prdetailbutton"></div>
+            </div>
+      </div>
+    </section>
+    <h2 class="title">
+      About
+    </h2>
+    <section id="about">
+    <p id="bio">
+      Lorem ipsum dolor sit amet, consectetur<br>
+      adipiscing elit, sed do eiusmod tempor<br>
+      incididunt ut labore et dolore magna<br>
+      aliqua. Ut enim ad minim veniam, quis<br>
+      nostrud exercitation ullamco laboris nisi<br>
+      ut aliquip ea eiusmod
+    </p>
+    <div type="image" id="cvbutton"></div>
+    <div id="techskills">
+      <div class="techbox" id="languages">
+        <h3 class="ulheader">Languages</h3>
+        <ul>
+          <li class="article">Javascript</li>
+          <li class="article">HTML</li>
+          <li class="article">CSS</li>
+        </ul>
+        <img id="langicon" src="images/ic_languages_mob.png">
+      </div>
+      <div class="techbox" id="frameworks">
+        <h3 class="ulheader">Frameworks</h3>
+        <ul>
+          <li class="article">NaN</li>
+          <li class="article">NaN</li>
+         <li class="article">Nan</li>
+        </ul>
+        <img id="frameicon" src="images/ic_framework.png">
+      </div>
+      <div class="techbox" id="skills">
+        <h3 class="ulheader">Skills</h3>
+        <ul>
+          <li class="article">Git</li>
+          <li class="article">GitHub</li>
+          <li class="article">Figma</li>
+        </ul>
+        <img id="skillicon" src="images/ic_skills.png">
+      </div>
+    </div>
+    </section>
+    <section id="contact">
+      <div id="contactbg">
+        <div id="socialicons">
+          <a href="https://stackoverflow.com"><img src="images/socialmedia-icons/ic_stackoverflow.png" alt="stackoverflow-icon"></a>
+          <a href="https://wwww.github.com/s2jack"><img src="images/socialmedia-icons/ic_github.png" alt="github-icon"></a>
+          <a href="https://twitter.com/1stGuler"><img src="images/socialmedia-icons/ic_twitter.png" alt="twitter-icon"></a>
+          <a href="https://www.linkedin.com/in/selcuk-guler/"><img src="images/socialmedia-icons/ic_linkedin.png" alt="linkedin-icon"></a>
+          <a href="https://medium.com/"><img src="images/socialmedia-icons/ic_medium.png" alt="medium-icon"></a>
+          <a href="https://www.angellist.com/"><img src="images/socialmedia-icons/ic_angellist.png" alt="angellist-icon"></a>

```

Comment:
- [ OPTIOANAL] Kindly make sure the social icon links are wrapperd in an unordered list to respect the semantic HTML rules. :recycle: